<launch>
  <node pkg="data_replay" name="data_replay_node" type="data_replay_node" output="screen" >

  <param name="imu_path" type="string" value="/midea_robot/ros2_ws/3/imu_2022_03_03_17_24_17.txt" />
  <param name="imu2_path" type="string" value="/midea_robot/ros2_ws/3/imu2_2022_03_03_17_24_17.txt" />
  <param name="odom_path" type="string" value="/midea_robot/ros2_ws/3/odom_2022_03_03_17_24_17.txt" />
  <param name="localization_path" type="string" value="/midea_robot/ros2_ws/3/gps_localization_2022_03_03_17_24_17.txt" />
  
  <param name="imu_replay" type="bool" value="true" />
  <param name="imu2_replay" type="bool" value="false" />
  <param name="odom_replay" type="bool" value="false" />
  <param name="localization_replay" type="bool" value="false" />
  
  <param name="rate" type="int" value="300" />
  
  </node>
</launch> 
